code/datums/components/turfs/transition_border.dm

```diff
@@ -9,6 +9,8 @@
  *
  * This way, we have *near* perfect native-like simulation with moves without
  * having to do anything too special.
+ *
+ * todo: multi-tile object support
```

Comment:
Therapist: What do we do when we feel overwhelmed?

Me: // todo:

Therapist: no

Reply:
ok tbf this is like an actually real issue

(it only works one way or the other and we need it noted down!!)